torch/overrides.py

```diff
@@ -883,34 +885,38 @@ def get_testing_overrides() -> Dict[Callable, Callable]:
         Tensor.__setstate__: lambda self, d: -1,
         Tensor.T.__get__: lambda self: -1,
         Tensor._backward_hooks.__get__: lambda self: -1,
-        Tensor._base.__get__: lambda self: -1,
-        Tensor._cdata.__get__: lambda self: -1,
-        Tensor.grad.__get__: lambda self: -1,
+        # type ignore[...] is required in the items below due to:
+        # torch/overrides.py:xyz: error: "int" has no attribute "__get__";
+        # torch/overrides.py:xyz: error: "bool" has no attribute "__get__";
+        # ...
+        Tensor._base.__get__: lambda self: -1,  # type: ignore[union-attr]
+        Tensor._cdata.__get__: lambda self: -1,  # type: ignore[attr-defined]
+        Tensor.grad.__get__: lambda self: -1,  # type: ignore[attr-defined]
         Tensor._grad.__get__: lambda self: -1,
         Tensor._grad_fn.__get__: lambda self: -1,
         Tensor.grad_fn.__get__: lambda self: -1,
-        Tensor._version.__get__: lambda self: -1,
+        Tensor._version.__get__: lambda self: -1,  # type: ignore[attr-defined]
```

Comment:
If mypy doesn't recognize this, we should file a missing annotation report.

Reply:
Actually, `Tensor._version` is already annotated and this `type: ignore` can be safely removed.

Reply:
Oh, in that case I'd expect all the `X.__get__` ignores can be removed. Please do so.

Reply:
Done!